kratos/includes/fill_communicator.h

```diff
@@ -90,6 +90,14 @@ class KRATOS_API(KRATOS_CORE) FillCommunicator
      */
     virtual void Execute();
 
+    /**
+     * @brief Execute the communicator fill
+     * This method is intended to perform the communicator filling In the current serial case it does nothing.
+     * For the parallel implementation see ParallelFillCommunicator.
+     * @param rModelPart DataCommunicator to be used for the partitioning
+     */
+    virtual void Execute(const DataCommunicator& rDataComm);
```

Comment:
hmmm, I thought you would add this to the constructor. What would be the use case for changing it in Execute? Does it work if we call with a different one?

Reply:
Works in either case, the one without the arg takes the default
I don't have a strong opinion, will change it

Reply:
I think I did it to not have to save the DataComm as a member
In any case I will change it